### PR TITLE
add calendars for NFK

### PR DESCRIPTION
**Pull request by kamilatwork**

### CODE REVIEW

File: atb_school_product_and_details.xml [hunk 1]
```diff
@@ -1602,5 +1602,718 @@
...
+          <DayTypeAssignment version="1" id="NFK:DayTypeAssignment:SchoolDayNFK2023-2024" order="1">
+            <OperatingPeriodRef ref="NFK:UicOperatingPeriod:SchoolDayNFK2023-2024"/>
+            <DayTypeRef version="1" ref="NFK:FareDayType:SchoolDayNFK2023-2024"/>
+          </DayTypeAssignment>
+        </dayTypeAssignments>
+      </ServiceCalendar>
+    </ServiceCalendarFrame>
+
+    <!-- Nordland kommune: Lurøy, Sømna 2023-2024 -->
+    <ServiceCalendarFrame version="1" id="NFK:ServiceCalendarFrame:SchoolDayNFKLuroySomna2023-2024">
```
Comment (Nisha1306) [inline atb_school_product_and_details.xml]: Guess we can skip having NFK in all of the calendar names, apart from the common calendar?

File: atb_school_product_and_details.xml [hunk 2]
```diff
@@ -1602,5 +1602,718 @@
...
+      <Name lang="eng">School year calendar for Nordland kommune</Name>
```
Comment (Nisha1306) [inline atb_school_product_and_details.xml]: _School year calendar for Luroy and Somna kommune_ ?

File: atb_school_product_and_details.xml (hunk 1 above)
Comment (kamilatwork) [reply atb_school_product_and_details.xml]: Good point. Changed.

File: atb_school_product_and_details.xml (hunk 2 above)
Comment (kamilatwork) [reply atb_school_product_and_details.xml]: Names changed as well. Thanks!

**Review by Nisha1306 — APPROVED** [review]
Dint verify all of the calendars but rest looks good to me

**Comment by Nisha1306** [comment]
🎆

**Pull request merged**